docs/plugins/pluggable_rule_engine.md

```diff
@@ -293,3 +292,68 @@ The best practice for using both `delay()` and `remote()` [depends on the use ca
 .. - Failover checking
 ..
 -->
+
+## Metadata Guard Rule Engine Plugin
+
+The Metadata Guard iRODS Rule Engine Plugin protects metadata AVUs with attributes containing certain prefixes from modification by users in an iRODS zone. This is useful for protecting metadata used by administrative systems in the iRODS zone such as the [Storage Tiering Capability](../capabilities/storage_tiering.md) to ensure that these processes are uninterrupted.
+
+### Configuration
+
+The Rule Engine Plugin config is set as metadata on the **zone collection** (e.g. `/tempZone`).
+Each option is explained below.
+```javascript
+{
+    // The list of strings that represent metadata that should be guarded.
+    // In this example, any metadata beginning with "irods::" will be treated special
+    // and require that the user be an administrator or classified as an editor depending
+    // on the configuration.
+    "prefixes": ["irods::"],
+
+    // Only administrators are allowed to modify metadata.
+    // This option supersedes the "editors" option.
+    "admin_only": true,
+
+    // The list of editors that can modify guarded metadata.
+    "editors": [
+        {
+            // The type of entity that is allowed to modify metadata.
+            // The following options are available:
+            // - "user"
+            // - "group"
+            "type": "group",
+
+            // The name of the iRODS entity.
+            // For remote users, you must include the zone (e.g. "rods#tempZone").
+            "name": "rodsadmin"
+        }
+    ]
+}
```

Comment:
how does one set this in metadata?  i don't remember... might need an example.

Reply:
They have to use `imeta` to attach the JSON as metadata.

Reply:
oh, so ...

`imeta add -C /tempZone editors '["type": "group", "name": "rodsadmin"]'`

literally a JSON array as a string

Reply:
show me editors

Reply:
oh, just more JSON *in* the JSON...

Reply:
apparently an example using all the parts would be useful :P

Reply:
See line 334.

Reply:
So... do we want a more complete example for the `imeta` usage example? What do we want it to demonstrate?

Reply:
yes, a second example showing the additional use of the editors section would be helpful.

Reply:
Added the example text to the `imeta` command and verified that it works.